src/qt/communityfundsuccessdialog.cpp

```diff
@@ -0,0 +1,45 @@
+#include "communityfundsuccessdialog.h"
+#include "ui_communityfundsuccessdialog.h"
+#include <QDialog>
+
+CommunityFundSuccessDialog::CommunityFundSuccessDialog(uint256 hash, QWidget *parent, CFund::CProposal* proposal) :
+    QDialog(parent),
+    ui(new Ui::CommunityFundSuccessDialog),
+    proposal(proposal),
+    prequest(0)
+{
+    ui->setupUi(this);
+    connect(ui->pushButtonClose, SIGNAL(clicked()), this, SLOT(reject()));
+
+    // Generate label for proposal
+    ui->label->setTextFormat(Qt::RichText);
+    ui->label->setText(QString::fromStdString("You can now view your proposal in the "
+                                                             "core wallet or <a href=\"https://www.navexplorer.com/community-fund/proposal/"
+                                              + hash.ToString() + "\">on the navexplorer</a>"));
```

Comment:
actually I have one more suggestion - the success dialog box appears immediately after creating a proposal or preq, it has a link to NAVexplorer but the proposal/preq needs to be broadcast into the next block so this link will be broken until the explorer updates. So I would suggest either remove the link or link to the general proposal/preq page rather than the specific hash

Reply:
Good point, I'll make this change tomorrow